package.json

```diff
@@ -6,7 +6,9 @@
   "type": "module",
   "files": [
     "app/javascript/**/*.js",
-    "plugin.js"
+    "app/views/**/*.{arb,erb,html,rb}",
+    "plugin.js",
+    "vendor/javascript/flowbite.js"
```

Comment:
You can remove this line and rebase from master branch now. I was going to change this anyway but to include all JS files in `vendor/javascript` since it should be available in the NPM package. I figured I'd do that as separate PR for a clearer release note.